.github/workflows/ci.yml

```diff
@@ -42,3 +42,12 @@ jobs:
       - name: Typescript
         id: typescript_test
         run: npm run typescript --if-present
+
+  automerge:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: fastify/github-action-merge-dependabot@v2.0.0
+        if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request' }}
```

Comment:
The if is not needed anymore: https://github.com/fastify/github-action-merge-dependabot#basic-example

Reply:
Thanks @mcollina, resolved with https://github.com/fastify/fastify-diagnostics-channel/pull/7/commits/a396e81deb10ba2e2864c3cd4db808acb6f02a72